packages/mui-material/src/Slider/useSlider.ts

```diff
@@ -687,6 +687,13 @@ export function useSlider(parameters: UseSliderParameters): UseSliderReturnValue
     };
   };
 
+  const cssWritingMode = React.useMemo(() => {
+    if (orientation === 'vertical') {
+      return isRtl ? 'vertical-rl' : 'vertical-lr';
+    }
+    return undefined;
+  }, [isRtl, orientation]);
```

Comment:
Do we really need `React.useMemo` here? It's handling just an if statement and `useMemo` itself probably does more computation than this.